### PR TITLE
add weekly changelog 2.211, 2.212, 2.213 and 2.214

### PR DESCRIPTION
**Pull request by zhaoying818**
### 检查项（Submitter checklist）

- [ ] 已添加 Issue
- [x] 已阅读[翻译规范](https://github.com/jenkinsci/localization-zh-cn-plugin/blob/master/specification.md)
- [x] [原文地址](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)

### Desired reviewers

@jenkins-infra/chinese-localization-sig

### CODE REVIEW

File: content/_data/changelogs/weekly.yml [hunk 1]
```diff
@@ -3730,6 +3730,88 @@
...
+  # pull: 4323 (PR title: Switch Release Drafter to GitHub Actions)
+  # pull: 4371 (PR title: Replace compare() implementations with call to Integer.compare or Long.compare)
+  # pull: 4403 (PR title: removed obsolete SpotBugs suppresses)
+  # pull: 4409 (PR title: Spotbugs: remove UMAC_UNCALLABLE_METHOD_OF_ANONYMOUS_CLASS from global exclude)
+  # pull: 4486 (PR title: Add Jenkins Changelog Generator to the GitHub Actions release drafter Pipeline)
+
+- version: '2.212'
+  date: 2020-01-05
+  banner: |-
+    <b>WARNING:</b> 此版本引入了严重的回归，请参阅 <a href="https://issues.jenkins-ci.org/browse/JENKINS-60644">JENKINS-60644</a>。
```
Comment (yJunS) [inline content/_data/changelogs/weekly.yml]: `引入的严重的回归` 有些不太通顺的感觉

Comment (zhaoying818) [reply content/_data/changelogs/weekly.yml]:
```suggestion
    <b>WARNING:</b> 此版本存在严重的问题，请参阅 <a href="https://issues.jenkins-ci.org/browse/JENKINS-60644">JENKINS-60644</a>。
```

**Comment by zhaoying818** [comment]
add weekly changelog 2.214

File: content/_data/changelogs/weekly.yml [hunk 2]
```diff
@@ -3730,6 +3730,185 @@
...
+    message: |-
+      内部: 允许使用 DescriptorVisibilityFilter 来过滤 UI 上的 View 属性。
+  - type: bug
+    category: bug
+    pull: 4271
+    issue: 42658
+    authors:
+    - ilpianista
+    message: |-
+      修复从节点机器机离线时 Agent API 中的空指针异常（例如检索代理版本或操作系统说明）。
```
Comment (yJunS) [inline content/_data/changelogs/weekly.yml]: 这句话有些不通顺

Comment (zhaoying818) [reply content/_data/changelogs/weekly.yml]:
```suggestion
      修复代理脱机时代理 API 中的空指针异常（例如查询代理版本或操作系统说明）。
```

**Comment by zhaoying818** [comment]
:1234:

**Pull request merged**